tests/integration/seccomp-notify.bats

```diff
@@ -83,8 +83,9 @@ function scmp_act_notify_template() {
 }
 
 # Test important syscalls (some might be executed by runc) work fine when handled by the agent. noNewPrivileges FALSE.
+# fcntl: https://github.com/opencontainers/runc/issues/4328
 @test "runc run [seccomp] (SCMP_ACT_NOTIFY important syscalls noNewPrivileges false)" {
-	scmp_act_notify_template "/bin/true" false '"execve","openat","open","read","close"'
+	scmp_act_notify_template "/bin/true" false '"execve","openat","open","read","close","fcntl"'
```

Comment:
Don't you need to modify the agent to handle this? Or was it generic enough?

Reply:
The issue was happening before connecting to the agent, so I guess this is enough for now?

Reply:
It seems it is handled, it just returns 0: https://github.com/opencontainers/runc/blob/81d63f265aadd01cd2ebc82e178e0da59baf38a6/contrib/cmd/seccompagent/seccompagent.go#L221-L225

Reply:
The problem is that we call `fcntl` before the agent gets the fd, so it can't respond to anything. This is the same problem as trying to use seccomp-notify for `write`.